presto-parser/src/main/java/io/prestosql/sql/SqlFormatter.java

```diff
@@ -812,6 +812,11 @@ protected Void visitCreateSchema(CreateSchema node, Integer context)
             builder.append(formatName(node.getSchemaName()));
             builder.append(formatPropertiesMultiLine(node.getProperties()));
 
+            if (node.getPrincipal().isPresent()) {
```

Comment:
I guess this should be mentioned before properties (possibly there are some missing tests).